src/main/java/cz/cvut/kbss/termit/config/OAuth2SecurityConfig.java

```diff
@@ -96,6 +86,17 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         return http.build();
     }
 
+    /**
+     * An attempt to replicate auth provider from HttpSecurity
```

Comment:
Please rewrite this comment. Why is this an attempt? The comment should give definitive information about what the method does.